src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -597,6 +621,18 @@ type WebAppBuilder() =
         let state = this.SystemIdentity (state)
         { state with SecretStore = KeyVault (External(Unmanaged name)) }
 
+    [<CustomOperation "use_extension">]
+    member this.UseExtension(state:WebAppConfig, name:string) =
+        let state = this.SystemIdentity (state) // REVIEW: Implied as required in https://www.visualstudiogeeks.com/devops/installing-aspnet-core-site-extensions-for-azure-app-service-using-arm
+        { state with SiteExtensions = { Name = name; Location = None } :: state.SiteExtensions }
+
+    // An initial attempt at handling location override. Then I noticed that we can also specify version, so decided
```

Comment:
Yes. Either that or we can use a builder method to create some object that's needed.